doc/level1.rst

```diff
@@ -726,19 +726,33 @@ keys instead.  The MUA MUST be capable of assembling these keys into
 an OpenPGP certificate (:rfc:`RFC 4880 "Transferable Public
 Key"<4880#section-11.1>`) that indicates these capabilities.
 
+Secret key protection at rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 The secret key material should be protected from access by other
 applications or co-tenants of the device at least as well as the
 passwords the MUA retains for the user's IMAP or SMTP accounts.
 
-Secret key protection at rest
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The MUA SHOULD NOT protect the secret key with a password. All
-encrypted outgoing messages MUST be signed, which would require the
-user to enter the password for both reading and sending mail. This
-introduces too much friction to become part of a routine daily workflow.
-Protection of the user's keys at rest and other files is achieved more
-easily and securely with full-disk encryption.
+The MUA MAY protect the secret key (and other sensitive data it has
+access to) with a password, but it SHOULD NOT require the user to
+enter the password each time they send or receive a mail.  All
+encrypted outgoing messages MUST be signed, so if the user has to
```

Comment:
This "MUST" kind of tripped me up more than once reading this sentence at different times. Can we change the sentence structure to a "Since", to indicate earlier on that this "MUST" is just leading up to something, rather than specifying that something MUST be one way or another? (arguably, it should actually be a "must" because of this, but that will trip other people up so better don't change that)